src/index.js

```diff
@@ -24,26 +24,15 @@ function render(
   const baseElement = customBaseElement || customContainer || document.body
   const container = customContainer || baseElement.appendChild(div)
 
-  const plugins = mountOptions.global?.plugins || []
-
-  if (store) {
-    const {createStore} = require('vuex')
-    plugins.push(createStore(store))
-  }
-
-  if (routes) {
-    const requiredRouter = require('vue-router')
-    const {createRouter, createWebHistory} =
-      requiredRouter.default || requiredRouter
-
-    const routerPlugin = createRouter({history: createWebHistory(), routes})
-    plugins.push(routerPlugin)
+  if (store || routes) {
+    console.warn(`Providing 'store' or 'routes' options is now deprecated.
+You need to create a router/vuex plugin and provide it through 'global.plugins'.
+Check out the test examples on GitHub for further details.`)
```

Comment:
> You need to create a router/vuex plugin

Is it technically considered a plugin or an instance? (Don't fully know the official lingo.)

Also, any thoughts about adding a link to the file? I think it would help people find the test example a lot more easily. I remember having a hard time finding VTL examples when I first tried it out. (Speaking of which, I still need to get to that docs thing. :eyes:) Hopefully it wouldn't make the logs too fat?

Reply:
> Is it technically considered a plugin or an instance?

Yeah, I think they should be called instances, good point. Fixing it.

> Also, any thoughts about adding a link to the file?

Thought about it but felt a bit "too much". Once docs thing is available at testing-library.com, we could then set a short link alias so the output isn't that verbose 👍

Reply:
Yeah, that's fair. :+1: